lib/Backends/BackendUtils.cpp

```diff
@@ -27,6 +27,10 @@ uint8_t *glow::collectConstants(
 
   // At compile time condense constants to a single block of memory.
   // This allows the graph to go away after compile time.
+  // If there are no constants return nullptr.
+  if (constantMaxSize == 0) {
+    return nullptr;
```

Comment:
please also check the free part.

Reply:
Free is ok, alignedFree just aliases free which is a no action when called on nullptr.